.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+### Things you may try first 
```

Comment:
This template works for bug reports, but there are also questions or feature requests, which probably do not need specific template, but people should be welcome to enter even such issues. I would appreciate something like this:

Please use this text as a template and replace text in the sections or remove the entire section if that does not apply to your issue. For example in case of question or feature request, just description with some example is probably fine. Also remember to use github's markup forma properly, especialy in case of output or code listing.